packages/xo-server-sdn-controller/src/index.js

```diff
@@ -96,7 +96,7 @@ class SDNController extends EventEmitter {
 
     this._overrideCerts = false
 
-    this._unsetApiMethod = null
+    this._unsetApiMethod = undefined
```

Comment:
Do you need these?

Reply:
Yes to unset the api method when the plugin is unloaded.

Reply:
I meant do you need to initialize the properties with `undefined`?